api/users.js

```diff
@@ -10,6 +10,7 @@ module.exports = function(app) {
 	app.delete("/auth", logout);
 	app.   get("/users", index);
 	app.  post("/users", create);
+	app.   get("/users/me", current_user);
```

Comment:
Why create a whole new route instead of just adding the "me" feature to read?

Reply:
I see how this is conflicting with the `read` route. We could have a user called `"me"`.

Maybe it would be safer to have a different route (such as `/user` in the singular) than having `me` inside `read`—which would force us to prohibit the use of `'me'` as a user in other places.

What approach do you prefer?

Reply:
The url should be "users/me" but the best way to do it, is to update the pre-existing read function to have this functionality. Then prevent create from allowing the username "me".